recipe/meta.yaml

```diff
@@ -18,13 +18,13 @@ build:
 requirements:
   host:
     - pip
-    - python
+    - python >=3.6
   run:
     - azure-core <2.0.0,>=1.0.0
     - cryptography >=2.1.4
     - msal <2.0.0,>=1.3.0
     - msal_extensions >=0.2.2,<0.3.0
-    - python
+    - python >=3.6
     - six >=1.6
```

Comment:
```python
    install_requires=[
        "azure-core<2.0.0,>=1.0.0",
        "cryptography>=2.1.4",
        "msal<2.0.0,>=1.6.0",
        "msal-extensions~=0.3.0",
        "six>=1.6",
    ],
```
https://github.com/Azure/azure-sdk-for-python/blob/0d70f5758f4d6083fc1deb57012c94e15d0f79e8/sdk/identity/azure-identity/setup.py#L74-L80